dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -1255,6 +1259,13 @@ Simulation::terminate() {
     fileConstraints.close();
   }
 
+  if (dumpFinalValues_) {
+    string finalValuesDir = createAbsolutePath("finalValues", outputsDirectory_);
+    if (!exists(finalValuesDir))
+      create_directory(finalValuesDir);
+    model_->printInitValues(finalValuesDir);
```

Comment:
Maybe we could pass the file name prefix as parameter so that they are named dumpFinalValues-<COMPONENT ID>.txt?

Reply:
This one is not visible in the nrt references